### PR TITLE
setup Address table to meet xNAL formatting

### PR DESCRIPTION
**Pull request by clocasto**
Implemented an Address table which meets xNAL formatting (#4).

### CODE REVIEW

**Comment by nrod80** [comment]
Looks good to me

File: server/db/models/address.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    locality: {
+        type: Sequelize.STRING,
+        allowNull: false
+    },
+    // Dependent Locality (unused)
+    dependent_locality: {
+        type: Sequelize.STRING
+    },
+    // Zip Code / Postal Code
+    postal_code: {
```
Comment (curiousYi) [inline server/db/models/address.js:50]: Requiring zip code seems counterintuitive if we allow international shipping.

File: server/db/models/address.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        type: Sequelize.STRING
+    },
+    // Sub premise (unused)
+    sub_premise: {
+        type: Sequelize.STRING
+    }
+
+}, {
+    getterMethods: {
+        name_line: function() {
```
Comment (curiousYi) [inline server/db/models/address.js]: I think this should be full_name

**Comment by curiousYi** [comment]
slight misnaming issue with the getter/setters see my line comment.

**Pull request merged**